src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -602,6 +602,7 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
         BigNumber.ROUND_UP,
       ),
       wallet: wallet ?? undefined,
+      pubKey: poolAssetAccountId ? fromAccountId(poolAssetAccountId).account : undefined,
```

Comment:
use `poolAssetAccountAddress`? should be the same for evm either way (pretty sure at least), but for consistency alonside `useAllowance`

Reply:
Nice catch, `getThorchainFromAddress` does exactly the same as this!